### PR TITLE
Fixes #1244 - align 'please login' message left

### LINKED ISSUE #1244: centered "please login" looks strange
screen shot 2016-12-16 at 3 37 01 pm

If you're logged out and look at an issues page, the please login message is centered -- and looks strange, IMO. We should maybe left-align it w/ the issue container?

(Another possible option is to move it to the right hand side up top in another column, w/ the G for GitHub thing, I imagine we'd want labels to go there as well.).

### PR DESCRIPTION
**Pull request by zoepage**
#1244 

r? @magsout && @miketaylr

### CODE REVIEW

File: webcompat/static/css/development/components/issue.css [hunk 1]
```diff
@@ -156,7 +156,7 @@
 /* Link to log in */
 .wc-Issue-login {
   margin-top: 2.5em;
-  text-align:center;
+  text-align:left;
```
Comment (miketaylr) [inline webcompat/static/css/development/components/issue.css]: Can we add a space between the prop and val here? (I know it was missing before...)

Comment (magsout) [inline webcompat/static/css/development/components/issue.css]: 👍

**Comment by magsout** [comment]
r+ 👍

File: webcompat/static/css/development/components/issue.css (hunk 1 above)
Comment (zoepage) [inline webcompat/static/css/development/components/issue.css]: added space and spaces to all other values in the css that were missing. :)

**Comment by miketaylr** [comment]
> added space and spaces to all other values in the css that were missing. :)

Cool, thanks! In the future a whitespace change like that can be its own commit (since the other whitespace changes have nothing to do with the commit message).

**Pull request merged**